lib/cli/cliUpdates.js

```diff
@@ -42,24 +45,30 @@ async function checkVersions() {
 
 async function performUpdate() {
   console.log(`Updating npm package from GitHub repository...`);
-  exec(`sudo npm install -g ${pkg.repository.url}`, (error, stdout, stderr) => {
+
+  // Exec output contains both stderr and stdout outputs
+  console.log(chalk.italic(`sudo npm install -g ${pkg.repository.url}`));
+  const updateOutput = await exec(`sudo npm install -g ${pkg.repository.url}`);
+  const updatedPkgVersion = await exec("silverfin --version");
+
+  if (updateOutput.stderr) {
+    console.log(`Error: ${updateOutput.stderr}`);
     console.log(
-      "Executed command",
-      chalk.italic(`sudo npm install -g ${pkg.repository.url}`)
+      `You can try running the following command: ${chalk.bold(
+        `sudo npm install -g ${pkg.repository.url}`
+      )}`
     );
-    if (error || stderr) {
-      const errorMesssage = error ? error.message : stderr;
-      console.log(`Error: ${errorMesssage}`);
-      console.log(
-        `You can try running the following command: ${chalk.bold(
-          `sudo npm install -g ${pkg.repository.url}`
-        )}`
-      );
-      console.log(`If that still fails, try updating NPM first.`);
-      return;
-    }
-    console.log(chalk.bold(`Silverfin CLI updated`));
-  });
+    console.log(`If that still fails, try updating NPM first.`);
+    return;
+  } else {
+    console.log(`--------------`);
+    console.log(updateOutput.stdout);
+    console.log(
+      chalk.bold(`Silverfin CLI updated to version ${updatedPkgVersion.stdout}`)
+    );
+  }
+
+  return updateOutput;
```

Comment:
Error are not catched correctly.

Before:

![Screenshot from 2023-09-06 15-13-49](https://github.com/silverfin/silverfin-cli/assets/99397124/b6e19353-d5e8-4014-af62-ad81b5de0562)

After update:

![Screenshot from 2023-09-06 15-13-31](https://github.com/silverfin/silverfin-cli/assets/99397124/43ceae7b-66cd-4618-9044-93e6dd768de3)

Reply:
Added try/catch block so error fails gracefully